File: app/src/main/java/com/cookplan/recipe_new/add_ingredients/EditRecipeIngredientsPresenter.java
package com.cookplan.recipe_new.add_ingredients;

import com.cookplan.models.Ingredient;

/**
 * Created by DariaEfimova on 21.03.17.
 */

public interface EditRecipeIngredientsPresenter {
    public void getAsyncIngredientList();

    public void removeIngredient(Ingredient ingredient);

    void onStop();
}

File: app/src/main/java/com/cookplan/recipe/edit/ingredients/EditRecipeIngredientsPresenter.kt
package com.cookplan.recipe.edit.ingredients

import com.cookplan.models.Ingredient

/**
 * Created by DariaEfimova on 21.03.17.
 */

interface EditRecipeIngredientsPresenter {

    fun getAsyncIngredientList()

    fun removeIngredient(ingredient: Ingredient)

    fun onDestroy()
}
